collector/lib/ConnScraper.cpp

```diff
@@ -190,6 +190,60 @@ int ReadHexBytes(const char* p, const char* endp, void* buf, int chunk_size, int
   return i;
 }
 
+struct ProcessInfo {
```

Comment:
We should be more consistent with the naming of the classes and their fields. They should be consistent with the protobufs. What is the difference between exe and comm? What is the purpose of having this struct and the class Process, which has the same fields. It feels redundant.

Reply:
I don't agree on this point. Protobuf messages are loaded with the semantics of processing information in the Stackrox context, while Collector internal objects represent Linux resources. For instance, a Process is a process, not the signal event.